Dockerfile

```diff
@@ -1,6 +1,5 @@
-FROM registry.access.redhat.com/ubi9/go-toolset:1.22.9-1741020486 AS builder
+FROM brew.registry.redhat.io/rh-osbs/openshift-golang-builder:rhel_9_golang_1.23 AS builder
```

Comment:
And we're back at using a private image in upstream as before #274 :disappointed: ... well, maybe not a big deal since only RH seems to be contributing to this repo anyway.

An alternative could be to use some well known public builder, e.g. `quay.io/projectquay/golang:1.23` which is a mirror of the official golang builder at docker hub.